package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "razor",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "nx serve",
+    "build": "nx build",
+    "test": "nx test",
+    "execute-husky": "npx husky install && npx husky add .husky/pre-commit \"npx --no-install lint-staged\" && npx husky add .husky/pre-commit-msg \"exec < /dev/tty && npx cz --hook || true\""
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-commit-msg": "exec < /dev/tty && npx cz --hook || true"
+    }
+  },
+  "private": true,
+  "dependencies": {
+    "commitizen": "^4.2.5",
```

Comment:
this must be a dev dependency

Reply:
Moved

related commit - dfe257ff30a3f79da26444aeb1fa2a7bf2deb312